5_People/career-operations.md

```diff
@@ -1,38 +1,44 @@
 # Operations Career
 
-You start by sitting at the support desk and answering basic support enquiries by understanding the users' questions and finding solutions for them in our documentation. Then you start taking on billing and similar more sensitive issues, followed by being able to handle technical issues as well. After this point you can switch over to the Technical career or continue into managerial support positions, finally taking over company management.
+You start at the tech support desk answering enquiries by understanding the users' questions and finding solutions. You advance by taking on harder technical issues and communicating regularly with developers. After this point you can switch over to the Technical career or continue into managerial support positions, finally taking over company management.
 
-## Customer Support Specialist
+## Tech Support I
 
-* Handles basic support questions, mostly by knowing where to find answers in our docs.
-* Usually less than 2 years of experience being in Customer Support.
-
-## Customer Support Associate
-
-* Write and improves support documentation.
-* Handles billing issues.
+* Handles a large range of support questions, many of them with technical issues to resolve.
+* Writes and improves support documentation.
 * Can write detailed bug reports for programmers to resolve an issue.
-* Usually at least 6 months of project domain knowledge.
+* Usually at least 2 years of tech support experience.
+* Salary.com role name for base salary: `Technical Support Analyst I`
 
-## Technical Support Specialist
+## Tech Support II
 
 * Familiarity with basic programming language features.
 * Writes and improves technical documentation.
 * Communicates with programmers about issues facing our users.
 * Designs and optimizes support workflows.
 * Usually at least 12 months of project domain knowledge.
+* Salary.com role name for base salary: `Technical Support Analyst II`
 
-## Operations Manager
+## Support Manager
 
-* Deep knowledge about the project and its users by regularly talking to users either in person or by phone. 
+* Manages other tech support Niteans.
 * Designs and maintains the structure of support documentation.
-* Capable of running and directing whole teams for smaller projects.
+* Communicates with programmers and project leads about the product development.
+* Salary.com role name for base salary: `IT Project Manager I`
+
+## Operations Manager
```

Comment:
The description of the roles is much closer to `Project Manager`. I.e. doing everything about a single project. Next step is doing operations on the company level, across multiple projects.